### PR TITLE
adds grue vulgaris! (planetstation creep mob)

### PR DESCRIPTION
**Pull request by ghost**
:cl: Armhulen
add: Grues! Always bring a light.
/:cl:

they're my jab at what nightmares tried to accomplish, to make maint a scary place to be. Nightmares took off as a struggle for survival without purpose because they are constantly hunted and have no objectives to do. As grues are not on this side of the coin, them having no objective is better.

also i like grues they're cool

WHAT I HAVE DONE:
grue jaunt
grue sight
grue sprites (lol)
fix up the grue walk
Turn night vision into an action button
WHAT I NEED DONE:
turning off carbon's 3x3 night vision around them when a grue is near. "It is pitch black. You are likely to be eaten by a grue."
Add the event to spawn these

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/nobodylooksatcommitts [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (JJRcop) [inline code/modules/mob/living/simple_animal/hostile/nobodylooksatcommitts]: yes I do

`git commit --allow-empty -m "Cool message here"`

Comment (Armhulen) [reply code/modules/mob/living/simple_animal/hostile/nobodylooksatcommitts]: shit

**Comment by ShadowDeath6** [comment]
I think it'd be cool to do oxy / some form of damage that's internal to the victim(heart attack?) rather than just brute, it doesn't sit right for a spooky scary darkness entity to just bite you to death, but I'm not sure how it'd be doable

**Comment by Armhulen** [comment]
both of those are really easy to do but not in spirit with the grue

**Comment by ShadowDeath6** [comment]
I assumed you had the lifeweb-esque grue in mind, ignore me then

**Comment by Armhulen** [comment]
lifeweb grues are kind of not really canon, so.

**Comment by drawsstuff** [comment]
Pretty sure he's aiming for the classic grue from Zork.

**Comment by arsserpentarium** [comment]
Wait,you can't explode grue?

**Comment by Repukan** [comment]
Grue Vulgaris?

In this station?

At this time?

Contained solely withing your dorms room?

**Comment by Buggy123** [comment]
make the spawn chance higher during night shifts

**Comment by RandomMarine** [comment]
\>another infini-jaunt antag

At least this one has a warning.

**Comment by Buggy123** [comment]
Nightmare really isn't that strong though, 9/10 times you get your ass handed to you because one taser before you can stab someone=dead

**Comment by Xhuis** [comment]
Neato! Having no/a vague sprite is probably good since they're, y'know, grues.

**Comment by Armhulen** [comment]
@Xhuis They have no sprite but i'll probably give them an indication that they're there for themselves so they don't get confused

**Comment by epochayur** [comment]
is this a user-controlled mob? or does it just wander around taking out assistants

**Comment by Xhuis** [comment]
I believe in `effects.dmi` there's a sprite for a black human silhouette, which seems like it'd fit well.

**Comment by Armhulen** [comment]
i already have sprites, xhuis. i'll pm you them on discord you'll love it

@epochayur player controlled that reminds me that i have to add these as an event

**Comment by Armhulen** [comment]
thanks for the thumbs up partner!

**Comment by arsserpentarium** [comment]
don't forget to add grue crate to cargo!

**Comment by ShadowDeath6** [comment]
How will this interact with night vision

**Comment by Ratvar** [comment]
"turning off carbon's 3x3 night vision around them when a grue is near."

**Comment by JJRcop** [comment]
There doesn't appear to be any animation  when a grue jaunts. Are they supposed to vanish imstantly?

**Comment by Armhulen** [comment]
@JJRcop lmao

**Comment by Qustinnus** [comment]
T G W E B  S O O N

**Comment by ShadowDeath6** [comment]
I meant nightvision as in NVG or other things that let you see 100% in the dark without needing light

**Comment by JJRcop** [comment]
@ShadowDeath6 This doesn't affect that, just the natural night vision.

**Comment by epochayur** [comment]
are people actually working on planetstation?

**Comment by ShadowDeath6** [comment]
>doesn't affect full night vision

I was asking how it interacts with FNV as a whole, it seems strange that you can just see a grue chasing you around, since it's intended to be "the monster that kills you in the dark"

**Comment by Armhulen** [comment]
You can't see a grue.

**Pull request closed without merging**